gradio/routes.py

```diff
@@ -409,9 +409,8 @@ async def run_predict(
             if hasattr(body, "session_hash"):
                 if body.session_hash not in app.state_holder:
                     app.state_holder[body.session_hash] = {
-                        _id: deepcopy(getattr(block, "value", None))
+                        _id: deepcopy(block)
```

Comment:
We should only deep copy the blocks that are inputs to the event, no? Doing a copy of every block will probably be really slow for big apps